artview/components/plot_grid.py

```diff
@@ -586,7 +586,7 @@ def cmapSelectCmd(self, cm_name):
 
     def toolZoomPanCmd(self):
         '''Creates and connects to a Zoom/Pan instance.'''
-        from .tools import ZoomPan
+        from .toolbox import ZoomPan
```

Comment:
I don't understand the need of changing tools to toolbox, and when there is not good reason for change, maintaining things as they are is a reason to not do it.